lib/uniris/oracles/oracle_server.ex

```diff
@@ -0,0 +1,84 @@
+defmodule Uniris.Oracles.OracleCronServer do
+  @moduledoc false
+
+  use GenServer
+
+  alias Uniris.TransactionChain.{
+    Transaction,
+    TransactionData
+  }
+
+  # Public
+
+  @spec start_link(mfa: mfa(), interval: pos_integer()) ::
```

Comment:
In other schedulers, CronTab intervals are used(i.e 0 * * * * *), so you can leverage it also to help the customization and avoid conflicts in the scheduling